modules/navigation/nav_region.cpp

```diff
@@ -103,7 +103,98 @@ void NavRegion::set_navigation_mesh(Ref<NavigationMesh> p_navigation_mesh) {
 	polygons_dirty = true;
 }
 
+Vector3 NavRegion::get_closest_point_to_segment(const Vector3 &p_from, const Vector3 &p_to, const bool p_use_collision) const {
+	RWLockRead read_lock(region_rwlock);
+
+	bool use_collision = p_use_collision;
+	Vector3 closest_point;
+	real_t closest_point_d = FLT_MAX;
+
+	for (const gd::Polygon &p : get_polygons()) {
```

Comment:
the function takes `p_from` and `p_to`  where `p` stands for point, so here I'd recommend using `polygon` instead of `p` to avoid confusion